app/models/decorators/mvi_user_decorator.rb

```diff
@@ -22,10 +22,11 @@ def create
       mvi: response
     }
     @user
-  rescue MVI::RecordNotFound => e
+  rescue MVI::RecordNotFound
     # TODO(AJD): add metric
     Rails.logger.error "MVI record not found for user: #{@user.uuid}"
-    raise Common::Exceptions::RecordNotFound, "User not in found MVI: #{e.message}"
+    @user.attributes = {}
+    # raise Common::Exceptions::RecordNotFound, "User not in found MVI: #{e.message}"
```

Comment:
👍 I agree with no longer raising RecordNotFound, but if it's not longer needed should we just remove the commented out code?